app-admin/webapp-config/webapp-config-1.55-r3.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1 prefix
+
+if [[ ${PV} = 9999* ]]
+then
+	EGIT_REPO_URI="https://gitweb.gentoo.org/proj/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://dev.gentoo.org/~ceamac/${CATEGORY}/${PN}/${P}.tar.bz2"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+fi
+
+DESCRIPTION="Gentoo's installer for web-based applications"
+HOMEPAGE="https://sourceforge.net/projects/webapp-config/"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="+portage"
+
+DEPEND="app-text/xmlto
+	sys-apps/gentoo-functions"
+RDEPEND="
+	portage? ( sys-apps/portage[${PYTHON_USEDEP}] )"
+
+python_prepare_all() {
+	# make the source from svn mirror the one in the tarball
```

Comment:
It did not install without this block.  I compared the git with the tarball and it seems it has a copy of itself in a directory named `webapp-config`.  Doing this fixed the problem.